edx_rest_framework_extensions/auth/jwt/tests/utils.py

```diff
@@ -33,14 +32,14 @@ def generate_asymmetric_jwt_token(payload):
     """
     Generate a valid asymmetric JWT token for authenticated requests.
     """
-    keys = jwk.KEYS()
+    import pdb; pdb.set_trace()
     serialized_keypair = json.loads(settings.JWT_AUTH['JWT_PRIVATE_SIGNING_JWK'])
-    keys.add(serialized_keypair)
+    key = RSAKey(serialized_keypair)
```

Comment:
Got the following error when I tried to run the given snippet.
```
>>> JWT_PRIVATE_SIGNING_JWK = (
...         '{"e": "AQAB", "d": "HIiV7KNjcdhVbpn3KT-I9n3JPf5YbGXsCIedmPqDH1d4QhBofuAqZ9zebQuxkRUpmqtYMv0Zi6ECSUqH387GYQF_Xv'
...         'FUFcjQRPycISd8TH0DAKaDpGr-AYNshnKiEtQpINhcP44I1AYNPCwyoxXA1fGTtmkKChsuWea7o8kytwU5xSejvh5-jiqu2SF4GEl0BEXIAPZs'
...         'gbzoPIWNxgO4_RzNnWs6nJZeszcaDD0CyezVSuH9QcI6g5QFzAC_YuykSsaaFJhZ05DocBsLczShJ9Omf6PnK9xlm26I84xrEh_7x4fVmNBg3x'
...         'WTLh8qOnHqGko93A1diLRCrKHOvnpvgQ", "n": "o5cn3ljSRi6FaDEKTn0PS-oL9EFyv1pI7dRgffQLD1qf5D6sprmYfWWokSsrWig8u2y0H'
...         'ChSygR6Jn5KXBqQn6FpM0dDJLnWQDRXHLl3Ey1iPYgDSmOIsIGrV9ZyNCQwk03wAgWbfdBTig3QSDYD-sTNOs3pc4UD_PqAvU2nz_1SS2ZiOwO'
...         'n5F6gulE1L0iE3KEUEvOIagfHNVhz0oxa_VRZILkzV-zr6R_TW1m97h4H8jXl_VJyQGyhMGGypuDrQ9_vaY_RLEulLCyY0INglHWQ7pckxBtI5'
...         'q55-Vio2wgewe2_qYcGsnBGaDNbySAsvYcWRrqDiFyzrJYivodqTQ", "q": "3T3DEtBUka7hLGdIsDlC96Uadx_q_E4Vb1cxx_4Ss_wGp1Lo'
...         'z3N3ZngGyInsKlmbBgLo1Ykd6T9TRvRNEWEtFSOcm2INIBoVoXk7W5RuPa8Cgq2tjQj9ziGQ08JMejrPlj3Q1wmALJr5VTfvSYBu0WkljhKNCy'
...         '1KB6fCby0C9WE", "p": "vUqzWPZnDG4IXyo-k5F0bHV0BNL_pVhQoLW7eyFHnw74IOEfSbdsMspNcPSFIrtgPsn7981qv3lN_staZ6JflKfH'
...         'ayjB_lvltHyZxfl0dvruShZOx1N6ykEo7YrAskC_qxUyrIvqmJ64zPW3jkuOYrFs7Ykj3zFx3Zq1H5568G0", "kid": "BTZ9HA6K", "kty"'
...         ': "RSA"}'
... )
>>>
>>>
>>> import json
>>> from authlib.jose.rfc7517.jwk import JsonWebKey
>>>
>>>
>>>
>>> serialized_keypair = JsonWebKey.import_key(json.loads(JWT_PRIVATE_SIGNING_JWK))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/umarkhan/edly/master/src/edx-drf-extensions/venv/lib/python3.8/site-packages/authlib/jose/rfc7517/jwk.py", line 43, in import_key
    return key_cls.import_key(raw, options)
  File "/Users/umarkhan/edly/master/src/edx-drf-extensions/venv/lib/python3.8/site-packages/authlib/jose/rfc7517/asymmetric_key.py", line 170, in import_key
    key = cls.import_dict_key(raw, options)
  File "/Users/umarkhan/edly/master/src/edx-drf-extensions/venv/lib/python3.8/site-packages/authlib/jose/rfc7518/rsa_key.py", line 105, in import_dict_key
    if 'd' in raw and not has_all_prime_factors(raw):
  File "/Users/umarkhan/edly/master/src/edx-drf-extensions/venv/lib/python3.8/site-packages/authlib/jose/rfc7518/rsa_key.py", line 119, in has_all_prime_factors
    raise ValueError(
ValueError: RSA key must include all parameters if any are present besides d
>>>
````

Reply:
This error message might be generated from this function

https://github.com/lepture/authlib/blob/master/authlib/jose/rfc7518/rsa_key.py#L112